frontend/src/Metamaps/Cable.js

```diff
@@ -1,5 +1,7 @@
 /* global $, ActionCable */
 
+import { indexOf } from 'lodash'
```

Comment:
you can run `node_modules/.bin/eslint frontend/src/Metamaps/Cable.js` on develop branch and you should see the issue